### PR TITLE
Fix bugs 19 04 23

### PR DESCRIPTION
**Pull request by darmhoo**

### CODE REVIEW

File: src/components/PaymentLog.vue [hunk 1]
```diff
@@ -1320,10 +1325,17 @@ export default {
     },
     getCalc() {
       this.watchCashPrice()
-
-      this.serial = ["product", "ap_cash_n_carry", "products"].includes(
-        this.salesLogForm.business_type_id?.slug
-      )
+      if (this.salesLogForm.business_type_id) {
+        if (
```
Comment (teebarg) [inline src/components/PaymentLog.vue:1329]: can't this work here??

this.serial = ["product", "ap_cash_n_carry", "products"].includes(
        this.salesLogForm.business_type_id?.slug
      )

Comment (teebarg) [reply src/components/PaymentLog.vue:1329]: The second if condition

**Comment by darmhoo** [comment]
It won't work that way. i am testing if a slug contains a string not if a slug is in an array. That is y the code broke initially

**Comment by teebarg** [comment]
> It won't work that way. i am testing if a slug contains a string not if a slug is in an array. That is y the code broke initially

It should work, check well

**Comment by teebarg** [comment]
> It won't work that way. i am testing if a slug contains a string not if a slug is in an array. That is y the code broke initially

Lets pull all the slug that meets the condition inside that array

**Comment by darmhoo** [comment]
done

**Pull request merged**